blueocean-dashboard/src/main/js/components/Branches.jsx

```diff
@@ -49,16 +56,16 @@ export default class Branches extends Component {
 
         return (
             <tr key={cleanBranchName} onClick={open} id={`${cleanBranchName}-${id}`} >
-                <td><WeatherIcon score={weatherScore} /></td>
-                <td onClick={open}>
+                <BranchCol><WeatherIcon score={weatherScore} /></BranchCol>
+                <BranchCol onClick={open}>
                     <LiveStatusIndicator result={result === 'UNKNOWN' ? state : result}
                       startTime={startTime} estimatedDuration={estimatedDurationInMillis}
                     />
-                </td>
-                <td>{cleanBranchName}</td>
-                <td><CommitHash commitId={commitId} /></td>
-                <td>{msg || '-'}</td>
-                <td><ReadableDate date={endTime} liveUpdate /></td>
+                </BranchCol>
+                <BranchCol>{cleanBranchName}</BranchCol>
+                <BranchCol><CommitHash commitId={commitId} /></BranchCol>
+                <BranchCol>{msg || '-'}</BranchCol>
+                <BranchCol><ReadableDate date={endTime} liveUpdate /></BranchCol>
                 { /* suppress all click events from extension points */ }
                 <td className="actions" onClick={(event) => stopProp(event)}>
```

Comment:
If the `onClick` handler is removed from the TR then the I believe this `onClick` can also be killed. To test it, try favoriting a branch and verify that Run Detail doesn't open.